src/universal/bin/eckeygen

```diff
@@ -2,4 +2,4 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR/..
-./bin/mantis -- eckeygen "$@"
+./bin/mantis-core -- eckeygen "$@"
```

Comment:
should be: `mantis`  (This was fixed here: https://github.com/input-output-hk/mantis/pull/671)